data/blog/developing-advancements-in-onchain-privacy.mdx

```diff
@@ -0,0 +1,224 @@
+announced in [July](/blog/bitcoin-grants-july-2023#payjoin-dev-kit) 2023,
+[August](/blog/josi-baker-receives-opensats-lts-grant) 2023, and
+[October](/blog/8th-wave-of-bitcoin-grants#citadel-tech) 2024, respectively:
+
+- [Payjoin](#payjoin-v2)
+
+- [Coinswap](#coinswap)
+
+- [Silent Payments](#silent-payments)
+
+The developers working on these projects are making meaningful impacts, with
+real-world implementations in place and ongoing improvements ahead. Let's take a
+closer look at how these projects are advancing and reinforcing Bitcoin's
+privacy and thus end-user security.
+
+---
+
+### Payjoin V2
+
+One of the most powerful tools for on-chain privacy is breaking the assumptions
+used by blockchain surveillance, which are based on various heuristics and
+transaction patterns.
+
+One such assumption is the common input ownership heuristic
+([CIOH](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic)), which
+suggests that all inputs in a transaction belong to the same entity. The CIOH is
+one of the most common methods to track payments and link addresses, effectively
+degrading the privacy—and thus security—of all users. Payjoin disrupts this
+heuristic, which in turn improves the privacy characteristics of the whole
+network. Even if only a small percentage of users use Payjoin as their default
+payment mechanism, the common input ownership heuristic can't be trusted
+anymore, making it a useless heuristic for transaction deanonymization. *"The
+main problem is that,"* as Francis Pouliot, CEO of [Bull
+Bitcoin](https://www.bullbitcoin.com/) pointed out, *"to make it happen, a lot of
+open-source software infrastructure was missing."*
+
+Version 1 of Payjoin ([Pay-to-EndPoint](https://en.bitcoin.it/wiki/PayJoin))
+disrupted the CIOH assumption by modifying how transactions are constructed.
+Instead of a straightforward sender-to-recipient payment, the recipient also
+contributes one of their inputs to the transaction. This process reduces the
+effectiveness of common tracking and surveillance methods in use today. However,
+the challenge with this version was that it required real-time coordination,
+meaning both sender and receiver had to be online at the same time, which is not
+practical at scale.
+
+Thanks in part to [OpenSats
+funding](/blog/bitcoin-grants-july-2023#payjoin-dev-kit), Payjoin V2 ([BIP
+77](https://github.com/bitcoin/bips/pull/1483)) launched in 2024, implementing a
+major improvement called asynchronous transactions. Due to this, Payjoin V2 no
+longer requires both sender and receiver to be online simultaneously, making
+private and secure transactions seamless for users. This enhancement makes it
+easier for mobile wallets and exchanges to integrate Payjoin without disrupting
+their existing user workflows, making privacy and security protections more
+accessible to a wider audience.
+
+In December 2024, Bull Bitcoin became the first bitcoin exchange to process
+Payjoin V2 withdrawals natively. This
```

Comment:
```suggestion
In December 2024, Bull Bitcoin Mobile became the first mobile wallet to enable
sending and receiving Async Payjoin. This
```

The exchange hasn't enabled deposits / withdrawals yet (2 weeks). The self-custody mobile wallet got the update.